src/sentry/static/sentry/app/views/performance/landing.tsx

```diff
@@ -178,18 +178,30 @@ class PerformanceLanding extends React.Component<Props, State> {
 
     // This is a temporary change for trends to test adding a default count to increase relevancy
     if (viewKey === FilterViews.TRENDS) {
+      const hasStartAndEnd = newQuery.start && newQuery.end;
+      if (!newQuery.statsPeriod && !hasStartAndEnd) {
+        newQuery.statsPeriod = DEFAULT_TRENDS_STATS_PERIOD;
+      }
       if (!newQuery.query) {
         newQuery.query =
           'count():>1000 transaction.duration:>0 p50():>0 avg(transaction.duration):>0';
       }
       if (!newQuery.query.includes('count()')) {
-        newQuery.query += 'count():>1000';
+        newQuery.query += (newQuery.query ? ' ' : '') + 'count():>1000';
```

Comment:
Would it be better to make use of `tokenizeSearch` and `.setTag(...)` here to add the new filters?